guides/common/modules/ref_capsule-ports-and-firewalls-requirements.adoc

```diff
@@ -47,11 +47,12 @@ ifndef::katello,satellite,orcharhino[]
 | 8443 | TCP | HTTPS | {ProjectName} | {SmartProxy} API | Smart Proxy functionality
 endif::[]
 ifdef::katello,satellite,orcharhino[]
-| 8443 | TCP | HTTPS | Client | Content Host registration | Initiation
+| 443 | TCP | HTTPS | Client | Content Host registration | Initiation
 
 Uploading facts
 
 Sending installed packages and traces
+| 8443 | TCP | HTTPS | Client | Content Host registration | Deprecated and only needed for Client hosts deployed before upgrades
```

Comment:
I'm torn on the wording, also partially because I'm unhappy with the *old* description (which you didn't write).

The purpose of the port in this scenario is to allow the client to talk to the RHSM API for obtaining and updating subscription data. This happens during *registration* but also afterwards (subscription-manager regularly updates subscriptions of the content host).

Now, with the changes we implemented, new hosts will register via `443` and store that in their configs, so that later communication also happens via `443`. However, existing hosts (those, that are already registered) don't get a configuration update and continue talking via `8443`.

Thus saying it's used for *registration* seems odd to me -- those hosts aren't doing any registration anymore as they are already registered. They do, however, keep on using that port for communication until reconfigured by the admin.

I wouldn't block this PR on improving this wording, but that improvement should happen!

Reply:
@evgeni If you have something specific you want me to change from "Content Host registration", let me know, otherwise we can keep as is and address in a future PR and we can merge this one. Up to you.

Reply:
@evgeni All I could think of was something related to "Content Host communication" but I'm not a subject matter expert so I defer to all of you who know this stuff better than me. :-)

Reply:
No idea, so :roller_skate: on